paradise_papers_search/static/js/search.js

```diff
@@ -2,72 +2,128 @@
  * Paradise Paper Search App
  *
  * @author: Junior Báez
- * @version: 0.1.0
+ * @version: 0.2.0
  * @requires ko - global KnowckoutJS object
  */
 (() => {
   'use strict';
 
   /**
-  *  Node types mapping
+  *  Nodes Settings metadata
   *  {
   *    '<node_type>': {
-  *        '<property_name>': '<Property Label>',
+  *        '<property_name>': ['<Property Label>', <boolDisplayTable>],
   *        ...
   *    },
   *   ...
   *  }
   */
-  const node_types = {
+  const nodes_settings = {
     'entity': {
-      'name': '',
-      'incorporation_date': 'Incorporation',
-      'jurisdiction': 'Jurisdiction',
-      'countries': 'Linked To',
-      'sourceID': 'Data From'
+      'name': ['', true],
+      'incorporation_date': ['Incorporation', true],
+      'jurisdiction': ['Jurisdiction', true],
+      'countries': ['Linked To', true],
+      'sourceID': ['Data From', true]
     },
     'officer': {
-      'name': '',
-      'countries': 'Linked To',
-      'sourceID': 'Data From'
+      'name': ['', true],
+      'countries': ['Linked To', true],
+      'sourceID': ['Data From', true]
     },
     'intermediary': {
-      'name': '',
-      'countries': 'Linked To',
-      'sourceID': 'Data From'
+      'name': ['', true],
+      'countries': ['Linked To', true],
+      'sourceID': ['Data From', true]
     },
     'address': {
-      'address': '',
-      'countries': 'Linked To',
-      'sourceID': 'Data From'
+      'address': ['', true],
+      'countries': ['Linked To', true],
+      'sourceID': ['Data From', true]
     },
     'other': {
-      'name': '',
-      'countries': 'Linked To',
-      'sourceID': 'Data From'
+      'name': ['', true],
+      'countries': ['Linked To', true],
+      'sourceID': ['Data From', true]
     }
   };
 
+  /**
+   * Representation of a Node
+   *
+   * @param node_type
+   * @param node_properties
+   */
+  class Node {
+
+    constructor(node_type, node_id_or_properties) {
+      this._node_type = node_type;
+      this._node_properties = {};
+      this._node_connections = ko.observableArray();
+
+      if (typeof node_id_or_properties === 'number') {
+        this._node_id = node_id_or_properties;
+      } else {
+        this._node_id = node_id_or_properties.node_id;
+        this._node_properties = node_id_or_properties;
+      }
+    }
+
+    /**
+     * Fetch connected nodes.
+     */
+    fetchConnections () {
+      $.getJSON(
+        '/fetch/node',
+        {
+          'id': this._node_id,
+          't': this._node_type
+        }
+      )
+      .done(nodeFetch => {
+        this._node_properties = nodeFetch.response.data.node_properties;
+        nodeFetch.response.data.node_connections.forEach(conns => {
```

Comment:
`conns` 🤔 
Is there a more specific name for this?

Reply:
It could be nodes_connected...